pyproject.toml

```diff
@@ -28,8 +29,7 @@ dependencies = [
   "tantivy==0.22.2",
   "tree-sitter==0.21.3",
   "tree-sitter-languages==1.10.2",
-  "univers==30.12",
-  "cvss==3.6"
```

Comment:
@zvigrinberg , please do not omit the cvss package, its required for the cvss feature

Reply:
It's presented, i just omitted the version, returning it...